src/main/java/org/opensearch/security/rest/DashboardsInfoAction.java

```diff
@@ -55,11 +55,18 @@
 import org.opensearch.security.user.User;
 
 import com.google.common.collect.ImmutableList;
+import org.opensearch.security.dlic.rest.support.Utils;
 
 public class DashboardsInfoAction extends BaseRestHandler {
+    private static final List<Route> legacyRoutes = ImmutableList.of(
```

Comment:
Why is it necessary to split routes into 2 list?

Reply:
Because we have two different set of routes in `DashboardInforAction`, one with `kibanainfo` and the other one with `dashboardsinfo`, the `kibanainfo` only works for `_opendistro` and the one only works with `_plugins`

Reply:
I still don't see why is it necessary to split routes into 2 lists that later are combined into one anyway and the benefit of using `addRoutesPrefix()`.

Reply:
As I said above for `kibanainfo` we **only** need  `/_opendistro/_security` prefix and for `dashbordainfo` we **only** need `/_plugins/_security` prefix, we need to have two separate lists, so we can call `addRoutesPrefix` for each of them with their special prefix.